test/test_orch.py

```diff
@@ -379,10 +380,8 @@ def test_main_with_exam_scores_with_duplicate_uniqnames_sent_on_different_runs(s
         new_transmitted_qs: QuerySet = transmitted_qs.filter(transmitted_timestamp__gt=current_dt)
         self.assertEqual(len(new_transmitted_qs), 3)
 
-        dup_subs_qs: QuerySet = new_transmitted_qs.filter(student_uniqname='hgranger')
-        self.assertEqual(len(dup_subs_qs), 2)
-        dup_subs: List[Submission] = list(dup_subs_qs)
-        self.assertEqual([dup_sub.attempt_num for dup_sub in dup_subs], [2, 3])
+        dup_subs: List[Submission] = list(new_transmitted_qs.filter(student_uniqname='hgranger').order_by('id'))
```

Comment:
The two records of "hgranger" in canvas_subs.json have same id "123457". Is that as desired?

Reply:
Yes, this is intentional and how Canvas works. Each attempt for an assignment has the same Canvas ID. This is why I needed to combine the ID with something else (first `attempt_num`, now `graded_timestamp`) to uniquely identify the `Submission`. More background is provided in issue #27.

Reply:
Thanks for the clarification!